x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/index_mappings.tsx

```diff
@@ -5,8 +5,55 @@
  * 2.0.
  */
 
-import React from 'react';
+import React, { useEffect } from 'react';
+
+import { useActions, useValues } from 'kea';
+
+import {
+  EuiCodeBlock,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiLink,
+  EuiPanel,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+
+import { DocumentsLogic } from './documents_logic';
+import { IndexNameLogic } from './index_name_logic';
 
 export const SearchIndexIndexMappings: React.FC = () => {
-  return <>Index Mappings</>;
+  const { indexName } = useValues(IndexNameLogic);
+  const { makeMappingRequest } = useActions(DocumentsLogic);
+  const { mappingData } = useValues(DocumentsLogic);
+
+  useEffect(() => {
+    makeMappingRequest({ indexName });
+  }, [indexName]);
+
+  return (
+    <EuiPanel>
+      <EuiFlexGroup>
+        <EuiFlexItem grow={2}>
+          <EuiCodeBlock language="json" isCopyable>
+            {JSON.stringify(mappingData, null, 2)}
+          </EuiCodeBlock>
+        </EuiFlexItem>
+        <EuiFlexItem grow={1}>
+          <EuiPanel grow={false} hasShadow={false} hasBorder>
+            <EuiTitle>
+              <h3>About index mappings</h3>
+            </EuiTitle>
+            <EuiText>
+              <p>
+                Index mappings tell all the bits of bacon where arrange themselves in the omlet.
```

Comment:
i18n

Reply:
I mean, I'd also eh...change this text ;)

Reply:
Yes— omelette is misspelled here 🤣.